cluster-provision/k8s/1.17/provision.sh

```diff
@@ -24,6 +32,37 @@ yum -y remove firewalld
 # Required for iscsi demo to work.
 yum -y install iscsi-initiator-utils
 
+# Install docker required packages.
+dnf -y install yum-utils \
+    device-mapper-persistent-data \
+    lvm2
+
+# Add Docker repository.
+dnf config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+
+# Install Docker CE.
+dnf install -y docker-ce --nobest
+
+# Create /etc/docker directory.
+mkdir /etc/docker
+
+# Setup docker daemon
+cat << EOF > /etc/docker/daemon.json
+{
+  "insecure-registries" : ["registry:5000"],
+  "log-driver": "json-file",
+  "exec-opts": ["native.cgroupdriver=systemd"]
```

Comment:
If you would like to add ipv6 support:
`
  "ipv6": true,
  "fixed-cidr-v6": "2001:db8:1::/64"
`

Reply:
This is compatible with ipv4 right ?

Reply:
yes